### PR TITLE
Add doc changes related to top down API development feature

### PR DESCRIPTION
**Pull request by chashikajw**
## Purpose
As $subject, this will add required doc changes related to Top-down approach to create managed API feature

## Fixes
- https://github.com/wso2/docs-apim/issues/5256

### CODE REVIEW

File: en/docs/integrate/develop/creating-artifacts/creating-an-api.md [hunk 1]
```diff
@@ -142,6 +150,46 @@ Follow the instructions given below to create a new [REST API]({{base_path}}/ref
             </tr>
         </table>
 
+    -   If you selected **Import API from API Manager** in the previous step, enter the following information:
+
+        <img src="{{base_path}}/assets/img/integrate/create_artifacts/new_api/import-api-dialog.png" width="500">
```
Comment (Mariangela) [inline en/docs/integrate/develop/creating-artifacts/creating-an-api.md:155]: Add the image width as a %

File: en/docs/integrate/develop/creating-artifacts/creating-an-api.md [hunk 2]
```diff
@@ -142,6 +150,46 @@ Follow the instructions given below to create a new [REST API]({{base_path}}/ref
...
+                <td>
+                    Password of the API Manager user.
+                </td>
+            </tr>
+            <tr>
+                <td>
+                    API Manager host
+                </td>
+                <td>
+                    Host Url of the API Manager.
```
Comment (Mariangela) [inline en/docs/integrate/develop/creating-artifacts/creating-an-api.md:187]:
```suggestion
                    Host URL of the API Manager.
```

File: en/docs/integrate/develop/creating-artifacts/creating-an-api.md [hunk 3]
```diff
@@ -142,6 +150,46 @@ Follow the instructions given below to create a new [REST API]({{base_path}}/ref
...
+                </td>
+            </tr>
+        </table>
+        After entering above values in the Import API from API Manager option wizard, click <b>List APIs</b> button and it will list down the APIs which are in WSO2 API Manager. Thereafter, choose one of the APIs from the API list.
```
Comment (Mariangela) [inline en/docs/integrate/develop/creating-artifacts/creating-an-api.md:191]:
```suggestion
        After entering the above values in the <b>Import API from API Manager</b> option wizard, click <b>List APIs</b>. The list of APIs that are in WSO2 API Manager appear. Thereafter, select one of the APIs from the API list.
```

File: en/docs/tutorials/develop-an-integration-with-a-managed-api.md [hunk 4]
```diff
@@ -15,7 +15,7 @@ The following diagram demonstrates how the integration components and API manage
 
 Let's work with the following user story in this tutorial.
 
-The API developer has a Swagger definition that requires the creation of an API to be managed and some integrations. The API developer creates a prototype API via the Control Plane (Publisher Portal), creates mock implementations, and tests the API. The integration developer picks up the API definition, implements the integrations and deploys it in the Micro Ingegrator server. Thereafter, the API developer adds the service endpoint using API Manager and publishes the prototyped API.
+The API developer need to create an API first and managed it before adding the backend server implementation. The API developer creates an API via the Control Plane (Publisher Portal), creates mock implementations, and tests the API. The integration developer picks up the API definition, implements the integrations and deploys it in the Micro Integrator server. Thereafter, the API developer adds the service endpoint using API Manager and publishes the created API.
```
Comment (Mariangela) [inline en/docs/tutorials/develop-an-integration-with-a-managed-api.md:18]:
```suggestion
The API developer needs to create an API first and managed it before adding the backend server implementation. The API developer creates an API via the Control Plane (Publisher Portal), creates mock implementations, and tests the API. The integration developer picks up the API definition, implements the integrations and deploys it in the Micro Integrator server. Thereafter, the API developer adds the service endpoint using API Manager and publishes the created API.
```

**Review by Mariangela — APPROVED** [review]

**Comment by Mariangela** [comment]
@chashikajw , please note my comments and suggestions.

File: en/docs/integrate/develop/creating-artifacts/creating-an-api.md [hunk 5]
```diff
@@ -39,6 +39,14 @@ Follow the instructions given below to create a new [REST API]({{base_path}}/ref
                 Select this option to import an existing REST API configuration (XML definition) that was created using WSO2 Integration Studio.
             </td>
         </tr>
+        <tr>
+            <th>
```
[the remark this answers is not in the document]
Comment (chashikajw) [reply en/docs/integrate/develop/creating-artifacts/creating-an-api.md:43]: Ack. Will change in the next PR along with the other screenshots

**Pull request merged**